vtexdocs/RapiDoc
# Fix/url

#### What is the purpose of this pull request?

To show the full endpoint URL in the top of the API Reference page.

#### What problem is this solving?

On some documentations part of the path that was common between for more than one endpoint was contained only in the server URL, so the endpoint path was incomplete.

#### How should this be manually tested?

Open any API Reference page and check if the URL contained in the component in the start of the page is complete.

#### Screenshots or example usage

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62757720/228538666-319a06bb-ae2f-4d8d-ad25-649ba50bb330.png) | ![image](https://user-images.githubusercontent.com/62757720/228538717-82da5312-0833-47c0-99d3-2f6e2c79c8c8.png) |

#### Types of changes
